src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -465,6 +477,21 @@ export default class geTemplateBuilderFormFields extends LightningElement {
         });   
     }
 
+    /*******************************************************************************
+     * @description Handles the custom event fired from the geTemplateBuilderFormSection child when
+     * there is a metadata error on the field such as a deleted field mapping or deleted source or
+     * target field.
+     * @param {object} event: object for the custom event
+     */
+    handleFieldMetadataError(event) {
```

Comment:
Wondering, do we not need any of the information brought over in the `event` object? We only need to be notified that an event was fired?

Reply:
@psadie-sfdo I've updated the custom event by renaming it to make it more generic to validation and have also passed a `showError` property there now to show/hide the metadata warning message. You can see that in commits 7b2337455bc9402966dfae573d297fd8f50b0118, 458061ffb35200af9f0952c4d358f9aee79e64ed, and 998a343f7a8656733b6e46854de4024af5001f66